### ISSUE TITLE
how to use without node?

### ISSUE THREAD

**Issue by EVERYTHINGING** [msfeldstein/interactive-shader-format-js#6]
Hey there, wondeful library. I love the format and how it works with VDMX.

I was able to run the server in the examples using budo however, I would like to use the library without node, locally with one js file. What is the best way to do this? Or is it not recommended as node is doing some sort of heavy lifting that would not work well on the client side?

Thank you

**Comment by msfeldstein** [comment]
Good idea.  If you pull to master, you should be able to run `npm run build-global` which will output build/isf.global.js and should add the classes to your window object when included.

If this doesn't run let me know.  Whats the latest packaging norms for non-browserify based distribution?

**Comment by msfeldstein** [comment]
also if this does work let me know

**Comment by EVERYTHINGING** [comment]
command doesn't work. says missing script: build-global.

I was able to get this to work locally by running `browserify examples/app.js > examples/app-build.js` and then swapping app.js for app-build.js in the examples/index.html. However, this still uses the require statements. I was wondering if there was a way to build this out similarly to requirejs optimization tools which concats all of the js into one file in the necessary order?

**Comment by msfeldstein** [comment]
do you see this line in your package.json?  That is the script its trying
to run:
https://github.com/msfeldstein/interactive-shader-format-js/blob/master/package.json#L11

**Comment by EVERYTHINGING** [comment]
Rad thank you. I would like to submit a pull request with a separate index-local.html and app-local.js which makes it easy to get going without any tinkering but... I am getting a `fatal: unable to access 'https://github.com/msfeldstein/interactive-shader-format-js.git/': The requested URL returned error: 403` when I try to push up my new branch...

**Comment by msfeldstein** [comment]
you can't push to my repo, you need to create a fork to your own account, then create a pull request

**Comment by EVERYTHINGING** [comment]
aye ok will do this later thank you

**Comment by MacroMachines** [comment]
Im curious about this as well.  I was trying to see if i could run this in CODA for ios on my ipad pro, and getting the console error:
"ReferenceError: Can't find variable: require
I am here"
I haven't yet used node.js / npm / browserify in practice, so i am assuming there are dependencies there, but not sure how to find the bridge toward working without them or including a script reference to deal with this.

It seems like it should be possible to slim this all down into portable JS, am i correct in that assumption?

@EVERYTHINGING did you happen to post your version you mention?

**Comment by jasonmf** [comment]
The ability to use this as an ES Module would be really nice.
